xbmc/video/VideoDatabase.cpp

```diff
@@ -4864,10 +4864,23 @@ void CVideoDatabase::SetVideoSettings(int idFile, const CVideoSettings &setting)
   }
 }
 
-void CVideoDatabase::SetArtForItem(int mediaId, const MediaType &mediaType, const std::map<std::string, std::string> &art)
+void CVideoDatabase::SetArtForItem(int mediaId,
+                                   const MediaType& mediaType,
+                                   const std::map<std::string, std::string>& art)
 {
-  for (const auto &i : art)
-    SetArtForItem(mediaId, mediaType, i.first, i.second);
+  if (mediaType == MediaTypeMovie)
+  {
+    // Take the hit of looking up the default version of the movie only once, instead of once per
+    // art piece.
+    CFileItem item{};
```

Comment:
No need for explicit init. `CFileItem item;` would be enough.

Reply:
yes